coordinator/src/routes/prescription/post-convert-message.ts

```diff
@@ -17,6 +17,7 @@ export default [
       (requestPayload: Bundle, request: Hapi.Request, responseToolkit: Hapi.ResponseToolkit) => {
         const isSmokeTest = request.headers["x-smoke-test"]
         const contentType = isSmokeTest ? CONTENT_TYPE_PLAIN_TEXT : CONTENT_TYPE_XML
+        request.log("info", "Building HL7V3 message")
```

Comment:
got `request.logger.info()` above and `request.log("info", )` here, maybe best to choose one? think i saw the `.logger.info` in the other pr